### ISSUE TITLE
Issue related to npm install

### ISSUE THREAD

**Issue by sakshikatolkar** [insidesherpa/JPMC-tech-task-2-PY3#397]
![Screenshot (34)](https://user-images.githubusercontent.com/67429864/85751644-60dea400-b728-11ea-889c-610bafbf1257.png)
I've tried all the steps as you mentioned in the guide but it's not working. I tried troubleshooting but it is showing error and i dont know what to do to go ahead. Please help me out with this.

**Comment by Ayushihelloworld** [comment]
Yes, facing the same problem Were you able to resolve it?

**Comment by Ayushihelloworld** [comment]
Hi, I got it. I was facing the same error but after reading the documentation, there were following things you had to do:
1. Install the Visual Studio Code package(link given in the slide).
2. You had to install node js in your system.
3. If you have done both and still getting the error, simply run npm start. It won't show errors and directly open the localhost:3000.

**Comment by sakshikatolkar** [comment]
After npm start , localhost:3000 is showing in the browser but it is
showing error (failed to compile), and a type error. What can i do?

**Comment by Ayushihelloworld** [comment]
In that case, use these 4 commands in order
npm install
npm install --ignore-scripts
npm audit fix
npm start

refer to this link:
https://github.com/insidesherpa/JPMC-tech-task-2-PY3/issues/386

**Comment by sakshikatolkar** [comment]
Yes it worked. Thank you!!

**Comment by sravanakalhari** [comment]
> In that case, use these 4 commands in order
> npm install
> npm install --ignore-scripts
> npm audit fix
> npm start
> 
> refer to this link:
> #386

thank you !!!!! it helped me

**Comment by sakshikatolkar** [comment]
Hey one more issue occurred in graph.tsx. Its giving compilation error. Can you please help me if yours worked properly.
![Screenshot (38)](https://user-images.githubusercontent.com/67429864/86083760-ba442b80-bab8-11ea-802c-2e23b6e26f27.png)

**Comment by Ayushihelloworld** [comment]
You have an extra apostrophe in front of distinct count due to which it is showing you the syntax error

**Comment by sakshikatolkar** [comment]
Yes i tried to do that but after removing that it shows another error like this.
![Screenshot (39)](https://user-images.githubusercontent.com/67429864/86090172-3f364180-bac7-11ea-8a83-4feefa35e456.png)

**Comment by shubh38** [comment]
where should i install the npm...i'm the windows user

**Comment by 165shadow11** [comment]
> In that case, use these 4 commands in order
> npm install
> npm install --ignore-scripts
> npm audit fix
> npm start
> 
> refer to this link:
> #386

This fixed my issue, thank you!!
